README.md

```diff
@@ -83,23 +113,63 @@ less /var/log/vmware/connectors/jira/jira-connector.log
 ### Updating
 
 The connector can be updated using yum. For example, to update the Jira connector from 1.0.0 to 1.0.1:
+
 ```
 yum upgrade jira-connector-1.0.1.noarch.rpm
 ```
+
 Updating will not touch your overriding `application.properties` in `/etc`. It is probably a good idea to take a glance at `/opt/vmware/connectors/jira/application.properties` after updating to verify that there aren't any new things in the config that you should override. If there are any new settings you wish to configure, make sure you do so in the `application.properties` in `/etc` so that future updates/uninstalls don't throw away your config.
 
-## Uninstalling
+### Uninstalling
 
 ```
 yum remove jira-connector
 ```
 
+## Fat Jars
+
+The connectors can also be built and run as [executable "fat" jars](https://docs.spring.io/spring-boot/docs/current/reference/html/howto-build.html#howto-create-an-executable-jar-with-maven).
+
+### Prerequisites
+
+* [Java 8](https://www.java.com/en/download/help/index_installing.xml)
+
+### Building
+
+Clone this repository or download as zip.
+Unzip the repo if you downloaded a zip.
+Use the below command from within the repository directory to build the fat JARs:
+
+```
+./mvnw clean install
+```
+
+### Running
+
+After building the JAR(s), you can run them with `java -jar`.  For example, here is a command to run the jira-connector:
+
+```
+java -server \
+     -XX:+UnlockExperimentalVMOptions \
```

Comment:
I would leave out the `XX` options. As well as adding clutter, it's too prescriptive.

Reply:
Yea, I'm not sure what the right answer is on this one.  I really wish the JVM defaults were better.

Since the docker container options are more reasonable, should I leave those in and just remove them from the fat-jar example?

Reply:
Yep.